src/services/device.service.ts

```diff
@@ -0,0 +1,31 @@
+import { db } from '../database';
+
+export interface Device {
+  userId: number;
+  deviceToken: string;
+}
+
+interface InsertDeviceResult {
+  userId: number;
+  deviceToken: string;
+}
+
+interface DeviceCreated {
+  userId: number;
+  deviceToken: string;
+}
+
+const addDevice = async (device: Device): Promise<DeviceCreated> => {
+  const result = await db.sql<InsertDeviceResult>('insert-devices', {
+    user_id: device.userId,
+    device_token: device.deviceToken,
+  });
+  return {
+    userId: result.rows[0].userId,
+    deviceToken: result.rows[0].deviceToken,
```

Comment:
Likewise, the right-hand side of these lines needs to use camel_case:

```suggestion
    userId: result.rows[0].user_id,
    deviceToken: result.rows[0].device_token,
```